tmdb-api/src/commonMain/kotlin/app/moviebase/tmdb/model/TmdbModel.kt

```diff
@@ -143,7 +143,7 @@ data class TmdbVideo(
 
 @Serializable
 data class TmdbImages(
-    @SerialName("id") val id: Int,
+    @SerialName("id") val id: Int? = null,
```

Comment:
It is the movie `id`, and it comes as `null` from the API when the response is appended to the movie details. :man_facepalming:

Reply:
🤦‍♂️